bin/qdbg-remote

```diff
@@ -33,6 +34,14 @@ class DebugCommandLineRemote inherits DebugLinenoiseCommandLine {
         opts.errlog = \wscErr();
         opts.debuglog = \wscDebug();
         url = opts.url;
+        if (opts.header) {
+            foreach string h in (opts.header) {
+                *list<string> rv = regex_extract(h, "^([a-zA-Z0-9\\-]+)=(.*)$");
```

Comment:
it is slower to use the `regex_extract()` function than `*list<string> rv = (h =~ x/^([a-zA-Z0-9\-]+)=(.*)$/);`